release_notes/ocp-4-2-release-notes.adoc

```diff
@@ -0,0 +1,890 @@
+
+[id="ocp-4-2-new-api-explorer"]
+==== New API Explorer
+
+You can now easily search and manage API resources in the *Explore API
+Resources* dashboard located at *Home* -> *Explore*.
+
+View the schema for each API and what parameters being supported, manage the
+instances of the API, and review the access of each API.
+
+[id="ocp-4-2-machine-autoscaler"]
+==== Machine Autoscaler
+
+You can now scale your cluster with the Machine Autoscaler. The Machine
+Autoscaler adjusts the number of Machines in the MachineSets being deployed in
+your cluster. Increase Machines when the cluster runs out of resources to
+support more deployments.  Any changes, such as the minimum or maximum number of
+instances, are immediately applied to the MachineSet that MachineAutoscalers
+target.
+
+[id="ocp-4-2-developer-perspective"]
+==== Developer Perspective
+
+The Developer perspective adds a developer-focused perspective to the web
+console. It provides workflows specific to developer use cases, such as creation
+and deployment of applications to {product-title} using multiple options. It
+provides a visual representation of the applications within a project, their
+build status, and the components and services associated with them,  enabling
+easy interaction and monitoring. It incorporates Serverless capabilities
+(Technology Preview) and the ability to create workspaces to edit your
+application code using Eclipse Che.
+
+[id="ocp-4-2-prometheus-queries"]
+==== Prometheus queries
+
+You can now run Prometheus queries directly in the web console.
+
+[id="ocp-4-2-general-web-console-updates"]
+==== General web console updates
+
+* The dashboard is redesigned with more metrics.
+* *Catalog* is moved.
+* Status of projects is now moved to the *Workloads* menu.
+* OperatorHub is now located under the *Operators* menu.
+* There is now support for chargeback. You can break down the reserved and used resources requested by applications.
+* There is now support for native templates.
+
+[id="ocp-4-2-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} 4.2 introduces the following notable technical changes.
+
+[discrete]
+[id="ocp-4-2-layers"]
+==== Builds maintain their layers
+
+In {product-title} 4.2, builds keep their layers by default.
+
+See
+xref:../builds/running-entitled-builds.adoc#builds-strategy-docker-squash-layers_running-entitled-builds[Running
+entitled builds] for more information.
+
+Ingress controller TLS 1.0 and 1.1 support is now disabled to match the
+Mozilla intermediate security profile.
+
+New and upgraded ingress controllers will no longer support these TLS versions.
+
+[discrete]
+[id="ocp-4-2-windows-builds-not-supported"]
+==== Builds on Windows
+
+Builds on Windows nodes are not supported.
+
+[id="ocp-41-deprecated-features"]
+=== Deprecated features
+
+[discrete]
+[id="ocp-4-2-deprecation-of-service-catalog"]
+
+In {product-title} 4.2, the Service Catalog, the Template Service Broker, the
```

Comment:
@shawn-hurley @jmrodri The `ServiceCatalog/TSB/ASB` components do deprecate in 4.2?